addons/sale_margin/models/sale_order.py

```diff
@@ -13,7 +13,13 @@ class SaleOrderLine(models.Model):
         "Margin (%)", compute='_compute_margin', store=True, groups="base.group_user", precompute=True, group_operator="avg")
     purchase_price = fields.Float(
         string='Cost', compute="_compute_purchase_price",
+<<<<<<< HEAD
         digits='Product Price', store=True, readonly=False, precompute=True,
+||||||| parent of 9c948b510083 (temp)
+        digits='Product Price', store=True, readonly=False,
+=======
+        digits='Product Price', store=True, readonly=False, copy=False,
+>>>>>>> 9c948b510083 (temp)
```

Comment:
```suggestion
        digits='Product Price', store=True, readonly=False, copy=False, precompute=True,
```

Reply:
Sorry for the pings the last two weeks, I was on holidays, I'll handle the conflict resolution ;)

Reply:
I guessed so :smile: